app/dashboard/admin.py

```diff
@@ -470,11 +470,19 @@ class PollsAdmin(admin.ModelAdmin):
 
 
 class QuestionsAdmin(admin.ModelAdmin):
-    list_display = ['id', 'poll', 'question_type', 'text']
-    raw_id_fields = ['poll']
+    list_display = ['id', 'poll', 'question_type', 'text', 'img']
+    raw_id_fields = ['poll', 'header']
     search_fields = ['question_type', 'text']
     inlines = [OptionsInline]
 
+    def img(self, instance):
+        header = instance.header
+        if not header or not header.image:
+            return 'n/a'
+        img_html = format_html('<img src={} style="max-width:30px; max-height: 30px">', mark_safe(header.image.url))
```

Comment:
Note to other reviewers - generally we probably wouldn't allow this with the `mark_save` tag but in this case its probably ok since only Admins should have the ability to set and add an image